keel-veto/src/main/kotlin/com/netflix/spinnaker/keel/veto/Veto.kt

```diff
@@ -57,9 +59,21 @@ interface Veto {
    * What's currently being vetoed
    */
   fun currentRejections(): List<String>
+
+  /**
+   * What's currently being vetoed for an app
+   */
+  fun currentRejectionsByApp(application: String): List<ResourceId>
```

Comment:
I added this method to support ui views in the future. It seemed important that we could easily understand what vetos are vetoing which resources within an app.

Reply:
I can see this rolled into the `ApplicationController` response as-is. An initial first-pass in deck (cc: @erikmunson) could just be changing the color of :managed: badge when vetoed (no idea if that's low effort or even viable, just brainstorming.)